app/views/developers/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_for @developer_search_form, url:'/developers/search',  method: :post do |f| %>
+ <%= f.select :programming_language_id, ProgrammingLanguage.pluck(:name, :id), prompt: 'Select Programming Language', name: 'Programming Language' %>
+ <%= f.select :language_id, Language.pluck(:code, :id), prompt: 'Select Language', name: 'Language' %>
+ <%= f.submit 'Search' %>
+<% end %>
+
+<ul>
+ <% if(@developers) %>
+  <% @developers.each do |developer| %>
+    <li><%= developer.email %></li>
```

Comment:
Can you show other columns as follows?
![2018-04-06 16 42 42](https://user-images.githubusercontent.com/1908717/38408688-95702da4-39b9-11e8-99af-3fcb1c95ce93.png)